### PR TITLE
Updated missing block tooltips for English language

### PR DESCRIPTION
**Pull request by shivanshxyz**
In the .conf files present inside /learnbot_dsl/blocksConfig , most of the tooltips of various blocks were missing for English. So I updated them

### CODE REVIEW

File: learnbot_dsl/blocksConfig/basic/Operators.conf [hunk 1]
```diff
@@ -4,74 +4,74 @@
     "category" : "operator",
     "name" : "+",
     "shape" : ["block3"],
-    "tooltip" : {"ES": "Realiza la operación suma de los bloques conectados", "EN": ""}
+    "tooltip" : {"ES": "Realiza la operación suma de los bloques conectados", "EN": "Perform the sum operation of the connected blocks"}
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/basic/Operators.conf]:
```suggestion
    "tooltip" : {"ES": "Realiza la operación suma de los bloques conectados", "EN": "Perform the add operation of the connected blocks"}
```

File: learnbot_dsl/blocksConfig/basic/Operators.conf [hunk 2]
```diff
@@ -4,74 +4,74 @@
...
 },
 
 {
     "type" : "operator",
     "category" : "operator",
     "name" : "-",
     "shape" : ["block3"],
-    "tooltip" : {"ES": "Realiza la operación resta de los bloques conectados", "EN": ""}
+    "tooltip" : {"ES": "Realiza la operación resta de los bloques conectados", "EN": "Perform the operation subtract the connected blocks"}
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/basic/Operators.conf]:
```suggestion
    "tooltip" : {"ES": "Realiza la operación resta de los bloques conectados", "EN": "Perform the subtract operation of the connected blocks"}
```

File: learnbot_dsl/blocksConfig/basic/Operators.conf [hunk 3]
```diff
@@ -4,74 +4,74 @@
...
+    "tooltip" : {"ES": "Realiza la operación división de los bloques conectados", "EN": "Perform the division operation of the connected blocks"}
 },
 
 {
     "type" : "operator",
     "category" : "operator",
     "name" : "=",
     "shape" : ["block3"],
-    "tooltip" : {"ES": "Iguala el bloque de la izquierda al bloque de la derecha", "EN": ""}
+    "tooltip" : {"ES": "Iguala el bloque de la izquierda al bloque de la derecha", "EN": "Match the block on the left to the block on the right"}
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/basic/Operators.conf]:
```suggestion
    "tooltip" : {"ES": "Iguala el bloque de la izquierda al bloque de la derecha", "EN": "Assign the block on the left to the block on the right"}
```

File: learnbot_dsl/blocksConfig/basic/Operators.conf [hunk 4]
```diff
@@ -131,7 +131,7 @@
     "name" : "not",
     "shape" : ["block3"],
     "languages" : {"ES": "No", "EN": "not"},
-    "tooltip" : {"ES": "Niega una comparación", "EN": ""}
+    "tooltip" : {"ES": "Niega una comparación", "EN": "Deny a comparison"}
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/basic/Operators.conf]:
```suggestion
    "tooltip" : {"ES": "Niega una comparación", "EN": "Not operation of the conected block/blocks"}
```

File: learnbot_dsl/blocksConfig/default/Motor.conf [hunk 5]
```diff
@@ -80,7 +80,7 @@
         },
         "tooltip": {
             "ES": "Mueve el robot con la velocida de avance y la rotacion indicada",
-            "EN": ""
+            "EN": "Move the robot with the speed of advance and the indicated rotation"
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/default/Motor.conf]:
```suggestion
            "EN": "Move the robot with the specified translation and rotation speed"
```

File: learnbot_dsl/blocksConfig/default/Motor.conf [hunk 6]
```diff
@@ -96,7 +96,7 @@
         },
         "tooltip": {
             "ES": "Para el robot",
-            "EN": ""
+            "EN": "For the robot"
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/default/Motor.conf]:
```suggestion
            "EN": "Stop the robot"
```

**Review by pilarbachiller — CHANGES_REQUESTED** [review]
Please, review the translation. I have made suggestions for some of them, but you should review them all.

**Comment by shivanshxyz** [comment]
Thank you so much for your suggestions 😊

**Pull request closed without merging**